docs/process/release.md

```diff
@@ -6,37 +6,48 @@ Example [Milestone](https://github.com/unicode-org/icu4x/milestone/5) and [Issue
 
 Over the last month before the release target date, we'll discuss the checklist and milestone progress in our weekly calls, and fine tune timing and scope.
 
-Once the release is complete, the assigned release driver will:
-
-* Run `cargo +nightly fmt -- --config=format_code_in_doc_comments=true --config=doc_comment_code_block_width=80` to prettify our docs
-* Run `cargo update` for each `Cargo.lock` file to update our CI to freshest dependencies
-* Verify that the milestone and checklist are complete
-* Verify with component owners that they're ready for release
-* Verify that `ffi/diplomat` depends on a released (not Git) version of Diplomat. Get it published (ask manishearth or sffc) otherwise.
-* Verify that we have acceptable FFI coverage (should be a part of the checklist issue)
-  * Verify that `ffi/diplomat/tests/missing_apis.txt` is empty. If not, check with the team that we are okay punting these to the next release.
-* Take a bird-eye view at:
-  * READMEs
-  * Documentation
-  * Coverage
-  * Performance / Memory / Size benchmarks
-  * Cargo.toml files
-    * Cargo.toml files need to specify versions for each entry in `dependencies`.
-    * Cargo.toml should not specify versions for each entry in `dev-dependencies`.
-    * Ensure that any new packages have suitable version numbers.
-    * Ensure that the Cargo.toml file includes a set of fields consistent with other ICU4X packages.
+
+## Pre-release checklist
+
+This is a checklist of things that should be done in the weeks leading to the release.
+
+* [ ] Verify that the milestone and checklist are complete
+* [ ] Verify with component owners that they're ready for release
+* [ ] Take a bird-eye view at:
+  * [ ] READMEs
+  * [ ] Documentation
+  * [ ] Coverage
+  * [ ] Performance / Memory / Size benchmarks
+  * [ ] Cargo.toml files
+    * [ ] All dependencies from the workspace should use `workspace = true` rather than their version number or path 
+    * [ ] Cargo.toml files need to specify versions for each non-workspace entry in `dependencies`
```

Comment:
do we want to use workspace dependencies for non-workspace dependencies as well? I've only done workspace dependencies so far because that's a low-hanging fruit

Reply:
We can give people the option

Reply:
I think we should do either all or nothing

Reply:
I think it's fine for super common deps but overkill for everything. Would support a gradual move towards this for most deps.